### DISCUSSION TITLE
fail_on_revert = true unexpected behaviour

### DISCUSSION THREAD

**Discussion by Ynyesto** [Cyfrin/foundry-full-course-cu#1125] [Q&A]
Hi, guys! As you can see in the screenshot below, fail_on_revert = true is not working as expected for me. It does only run once, but it shows as passed, when the run clearly failed (and hence stopped). I saw a discussion on [GitHub](https://github.com/foundry-rs/foundry/issues/5726), so it must be a known Foundry issue, but I haven't found a solution. Has this happened to any of you? If so, how did you solve it? Thanks in advance!

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/33223441/cc4855a9-a59b-4099-adb2-819835c01c0a)

**Comment by 0xusmanf** [comment]
According to the discussion you mentioned, this issue was resolved 3 weeks ago. Try again after running `foundryup` command to update your foundry to the latest version.

**Reply by Ynyesto** [reply]
Yes! I did that and it worked, thanks!!
